package.json

```diff
@@ -66,15 +91,39 @@
     ],
     "keybindings": [
       {
-        "key": "alt+c",
-        "command": "clay.showTopLevelForm"
+        "key": "ctrl+alt+a ctrl+alt+f",
+        "command": "clay.file"
+      },
+      {
+        "key": "ctrl+alt+a ctrl+alt+q",
+        "command": "clay.fileQuarto"
+      },
+      {
+        "key": "ctrl+alt+a ctrl+alt+r",
+        "command": "clay.fileRevealJs"
+      },
+      {
+        "key": "ctrl+alt+a ctrl+alt+c",
+        "command": "clay.currentForm"
+      },
+      {
+        "key": "ctrl+alt+a q c",
```

Comment:
Does this work? Would be awesome if it did!

Reply:
OMG, it works! This changes things, I think.

Reply:
yes, it works.